pyproject.toml

```diff
@@ -20,6 +20,7 @@ classifiers = [
     "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
```

Comment:
So we report that we support it in the package description, but you do not want to add a README shield?

I wonder if we should not report that we support 3.12 until CPLEX catches up or we remove it in favor of a replacement cut-finding package

Reply:
> So we report that we support it in the package description, but you do not want to add a README shield?

Actually, the README shield is generated these days by querying the latest released version on pypi, so this PR (plus a release) will result in the shield being updated:

https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/blob/a666e696486b58798084b231eb4ccc76640b5a64/README.md#L7

> I wonder if we should not report that we support 3.12 at all until CPLEX catches up or we remove it in favor of a replacement cut-finding package

I think it's fine to say we support it in terms of the Trove classifiers, because we do.  Partial functionality works, and full functionality will work once there are cplex wheels.  But yeah, reasonable people could have different takes on this.  I do think it's worth pointing out that one should currently expect only partial functionality under Python 3.12, and this is what the note in INSTALL.rst is for.

Long term it'd of course be nice to remove the current cplex stuff in favor of #471.